packages/react-query-sdk/src/notifications/services/ContactPointsService.ts

```
@@ -0,0 +1,1209 @@
+  UseSuspenseInfiniteQueryOptions,
+  UseSuspenseInfiniteQueryResult,
+  UseSuspenseQueryOptions,
+  UseSuspenseQueryResult
+} from "@tanstack/react-query";
+
+import type { paths } from "../schema";
+export interface ContactPointsService {
+  createContactPoint: {
+    /**/
+    getMutationKey(
+      parameters: DeepReadonly<CreateContactPointParameters> | void
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)